File: docs/fundamentals/runtime-libraries/snippets/System/String/IsNullOrEmpty/csharp/isnullorempty1.cs
using System;

public class Example1
{
    public static void Main()
    {
        // <Snippet1>
        bool TestForNullOrEmpty(string s)
        {
            bool result;
            result = s == null || s == string.Empty;
            return result;
        }

        string s1 = null;
        string s2 = "";
        Console.WriteLine(TestForNullOrEmpty(s1));
        Console.WriteLine(TestForNullOrEmpty(s2));

        // The example displays the following output:
        //    True
        //    True
        // </Snippet1>
    }
}

File: docs/fundamentals/runtime-libraries/snippets/System/String/IsNullOrEmpty/fsharp/isnullorempty1.fs
namespace IsNullOrEmpty
open System

module IsNullOrEmpty1 =

    // <Snippet1>
    let testForNullOrEmpty (s: string): bool =
        s = null || s = String.Empty

    let s1 = null
    let s2 = ""

    printfn "%b" (testForNullOrEmpty s1)
    printfn "%b" (testForNullOrEmpty s2)

    // The example displays the following output:
    //    true
    //    true
    // </Snippet1>
